modules/cudf/src/series/numeric.ts

```diff
@@ -1407,6 +1407,82 @@ export abstract class NumericSeries<T extends Numeric> extends Series<T> {
     return this._process_reduction(skipna, memoryResource)._col.var(ddof, memoryResource);
   }
 
+  /**
+   * Return Fisher’s unbiased kurtosis of a sample.
+   * Kurtosis obtained using Fisher’s definition of kurtosis (kurtosis of normal == 0.0). Normalized
+   * by N-1.
+   *
+   * @param skipna Exclude NA/null values. If an entire row/column is NA, the result will be NA.
+   * @param memoryResource The optional MemoryResource used to allocate the result Series's device
+   *   memory.
+   * @returns The unbiased kurtosis of all the values in this Series.
+   * @example
+   * ```typescript
+   * import {Series} from '@rapidsai/cudf';
+   * const a = Series.new([1, 2, 3, 4]);
+   *
+   * a.kurtosis() // -1.1999999999999904
+   * ```
+   */
+  kurtosis(skipna = true, memoryResource?: MemoryResource) {
+    if (this.length == 0 || (this.hasNulls && !skipna)) { return NaN; }
+
+    const data = this._process_reduction(skipna, memoryResource);
+
+    const n = data.length;
+    if (n < 4) { return NaN; }
+
+    const V = data.var(skipna, 1, memoryResource);  // ddof = 1
+    if (V == 0) { return 0; }
+
+    const mu = data.mean(skipna, memoryResource);
+
+    const m4 =
+      Number(data.sub(mu, memoryResource).pow(4, memoryResource).sum(skipna, memoryResource)) /
+      (V ** 2);
+
+    // This is modeled after the cudf kurtosis implementation, it would be
+    // nice to be able to point to a reference for this specific formula
+    const a = (n * (n + 1)) / ((n - 1) * (n - 2) * (n - 3)) * m4;
+    const b = ((n - 1) ** 2) / ((n - 2) * (n - 3));
+
+    return a - 3 * b;
+  }
+
+  /**
+   * Return unbiased Fisher-Pearson skew of a sample.
+   *
+   * @param skipna Exclude NA/null values. If an entire row/column is NA, the result will be NA.
+   * @param memoryResource The optional MemoryResource used to allocate the result Series's device
+   *   memory.
+   * @returns The unbiased skew of all the values in this Series.
+   * @example
+   * ```typescript
+   * import {Series} from '@rapidsai/cudf';
+   * const a = Series.new([1, 2, 3, 4, 5, 6, 6]);
+   *
+   * a.skew() // -0.288195490292614
+   * ```
+   */
+  skew(skipna = true, memoryResource?: MemoryResource) {
+    if (this.length == 0 || (this.hasNulls && !skipna)) { return NaN; }
+
+    const data = this._process_reduction(skipna, memoryResource);
```

Comment:
This seems like a poorly named function. It's basically `this.nansToNulls().dropNulls()` or float series and `this.dropNulls()` for everything else. Should we consider letting `nansToNulls` exist on integral series, but be a no-op? 

This is certainly what Python cudf does:
```
In [45]: a = cudf.Series([1,2, 3])

In [47]: a.nans_to_nulls()
Out[47]: 
0    1
1    2
2    3
dtype: int64
```

Reply:
yep, I think that might be more consistent. +1 for a no-op `nansToNulls` operation for integral.

Reply:
Agreed, seems reasonable.

Reply:
left that for a future issue/PR https://github.com/rapidsai/node-rapids/issues/231